src/kibana/plugins/vis_types/vislib/editors/tile_map.html

```diff
@@ -11,9 +11,12 @@
 </div>
 
 <div class="vis-option-item">
-  </br>
   <label>
-    <input type="checkbox" value="{{isDesaturated}}" ng-model="vis.params.isDesaturated" name="isDesaturated" ng-checked="vis.params.isDesaturated">
+    <input type="checkbox"
+    name="isDesaturated"
+    value="{{isDesaturated}}"
```

Comment:
I know this is a holdover from the old code, but do you actually need value or ng-checked here? Angular should take care of both of these based on the ng-model.